### PR TITLE
Minimum spanning tree procedural

### PR DESCRIPTION
**Pull request by Caspersn2**

### CODE REVIEW

File: benchmarks/spanning_tree/procedural_c#/Program.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+			var (weight, edges) = computeMinspanTree();
+
+			Console.WriteLine("Total weight: " + weight);
+			Console.WriteLine("Total Edges: " + edges);
+		}
+
+		public static void ReadFileToHeap()
+		{
+			var lines = System.IO.File.ReadAllLines("benchmarks/spanning_tree/graph.csv");
+			var i = 0;
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/procedural_c#/Program.cs:34]:
```suggestion
```

File: benchmarks/spanning_tree/procedural_c#/Program.cs [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+					size++;
+					totalWeight += current.Weight;
+					totalEdges++;
+				}
+			}
+			return (totalWeight, totalEdges);
+		}
+
+		public static int[] vertexGroups;
+		public static int uniounFind(int node)
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/procedural_c#/Program.cs:64]: union. :) Skal rettes i hele filen

File: benchmarks/spanning_tree/procedural_f#/Program.fs [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    heap.array.[0] <- heap.array.[heap.size-1]
+    heap.size <- heap.size - 1
+    heapify 0
+    edge
+
+//------ End of heap
+
+let readFileToHeap() =
+    let lines = System.IO.File.ReadAllLines("benchmarks/spanning_tree/graph.csv");
+    let mutable i = 0
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/procedural_f#/Program.fs:86]:
```suggestion
```

File: benchmarks/spanning_tree/procedural_f#/Program.fs [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    for line in lines do
+        let elms = line.Split(',')
+        insert { Edge.Start = Int32.Parse(elms.[0]); Edge.End = Int32.Parse(elms.[1]); Edge.Weight = Int32.Parse(elms.[2]) }
+
+
+
+let vertexGrups: int array = Array.create (6005 + 1) -1
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/procedural_f#/Program.fs:93]: vertexGroup :)

File: benchmarks/spanning_tree/procedural_f#/Program.fs [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+let rec uniounFind (node: int) =
```
Comment (AnneEjsing) [inline benchmarks/spanning_tree/procedural_f#/Program.fs:94]: Union. :)

**Review by AnneEjsing — APPROVED** [review]

**Review by AnneEjsing — APPROVED** [review]

**Review by Nitram1995 — APPROVED** [review]
Ud over, at der stadig er hard coded ting, så ser det fint ud

**Pull request merged**